### DISCUSSION TITLE
System setups

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#103] [General]
Thanks to @foretspaisibles we have a script to help share our setups: `/development/describe`.

Here is mine:
```
Uname: Darwin
System: Undescriptible System
Architecture: x86_64
Date: Mon Sep 12 08:47:54 HST 2022
SBCL: /usr/local/bin/sbcl (SBCL 2.2.6)
Brew: /usr/local/bin/brew
MacPorts: :NOT-FOUND
QuickLisp: To load "closer-mop":
  Load 1 ASDF system:
    closer-mop
; Loading "closer-mop"

To load "trivial-main-thread":
  Load 1 ASDF system:
    trivial-main-thread
; Loading "trivial-main-thread"

To load "trivial-backtrace":
  Load 1 ASDF system:
    trivial-backtrace
; Loading "trivial-backtrace"

To load "cffi":
  Load 1 ASDF system:
    cffi
; Loading "cffi"
............
To load "cl-opengl":
  Load 1 ASDF system:
    cl-opengl
; Loading "cl-opengl"
..
To load "cl-glu":
  Load 1 ASDF system:
    cl-glu
; Loading "cl-glu"

To load "cl-glfw3":
  Load 1 ASDF system:
    cl-glfw3
; Loading "cl-glfw3"

To load "kons-9":
  Load 1 ASDF system:
    kons-9
; Loading "kons-9"
..................................................
............
T
```

**Comment by foretspaisibles** [comment]
The tool is meant as an easy way for developers and users to share their system's detail in an effortless standard way. The current set of details is just a short list of “usual suspects” and has to be extended as required. Anyone can just reach me for support, if required.

Given the content of other discussions, we could need to list CFFI, GLFW, OpenGL Frameworks paths and versions there. Should we work out a list of useful additional details and then figure out how to find them out?

That's interesting already as I see the Quicklisp script produces some garbage output. (Probably from :silent t) and I'look into this.

**Comment by istewart-jinzoningen** [comment]
Here's my output from my desktop PC:

Uname: Linux
System: Gentoo  
Architecture: x86_64
Date: Sat Feb 25 06:37:54 PM PST 2023
SBCL: /usr/bin/sbcl (SBCL 2.3.1)
Brew: :NOT-FOUND
MacPorts: :NOT-FOUND

; file: /home/istewart/quicklisp/quicklisp/dist.lisp
; in: DEFUN CONFIG-FILE-INITARGS
;     (INTERN (FUNCALL QL-DIST::*INITARG-CASE-CONVERTER* STRING) 'KEYWORD)
; 
; note: deleting unreachable code
; 
; compilation unit finished
;   printed 1 note
QuickLisp: T


I am trying to keep up with SBCL monthly point releases as they come out, and I'm running an RTX 3080 with the Nvidia binary Linux drivers on this machine. Feel free to ping me if additional testing with either of those features seems useful.
